code/game/gamemodes/gameticker.dm

```diff
@@ -465,6 +465,7 @@ var/global/datum/controller/gameticker/ticker
 
 /datum/controller/gameticker/proc/declare_completion()
 
+	nologevent = 1 //end of round murder and shenanigans are legal; there's no need to jam up attack logs past this point.

```

Comment:
Hm. Roundend should really be called on round end, not just for mutiny. 

But alright, this isn't terrible anyways, just a note.